jenkins/deploy.sh

```diff
@@ -97,7 +97,7 @@ TOOL_FPATH="deployjars/$TOOL_ART_ID-$TOOL_ART_VER"
 TOOL_DOC_JARS="-Dsources=${TOOL_FPATH}-sources.jar -Djavadoc=${TOOL_FPATH}-javadoc.jar"
 $DEPLOY_CMD -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \
             $TOOL_DOC_JARS \
-            -Dfile=$TOOL_FPATH.jar -DpomFile=${TOOL_PL}/pom.xml
+            -Dfile=$TOOL_FPATH.jar -DpomFile=${TOOL_PL}/dependency-reduced-pom.xml
```

Comment:
@NvTimLiu anything else we would need to update for tools jar deploy?

Reply:
actually specifically for the nightly do we need to do anything to the scripts?  @pxLi  as well

Reply:
We need also specify `-DpomFile=${TOOL_PL}/dependency-reduced-pom.xml` for the nightly deploy : 

https://github.com/NVIDIA/spark-rapids/blob/branch-22.06/jenkins/spark-nightly-build.sh#L119

Reply:
Thanks @NvTimLiu. I have updated nightly deploy script.